manifests/r/rocksdanister/LivelyWallpaper/1.6.0.4/rocksdanister.LivelyWallpaper.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: rocksdanister.LivelyWallpaper
+PackageVersion: 1.6.0.4
+Installers:
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: neutral
```

Reply:
That's impossible.. its x86 build.

Reply:
Didn't notice you were the dev, ignore that one, I'll report a bug if my eyes wasn't going sideways 🙄

Reply:
![image](https://user-images.githubusercontent.com/15158490/120887532-bb98a500-c5f3-11eb-8eb4-85e2d98a9a7b.png)
SEE I'M NOT CRAZY! >:C

Reply:
Is it some bug? taskmgr shows livelywpf.exe as 32bit right? 🤔

Reply:
> Is it some bug with the tool.. taskmgr shows livelywpf.exe as 32bit right? 🤔

Indeed it does, might be a issue how Windows sees x86 user installed programs then